tests/foreman/ui/test_reporttemplates.py

```diff
@@ -345,3 +403,60 @@ def test_negative_nonauthor_of_report_cant_download_it(session):
     :expectedresults: Report can't be downloaded. Error.
     :CaseImportance: High
     """
+
+
+@tier3
+def test_positive_generate_entitlements_reports_multiple_formats(session, content_setup):
+    """Generate reports using the Entitlements template in html, yaml, json, and csv format.
+
+        :id: b268663d-c213-4e59-8f81-61bec0838b1e
+
+
+        :setup: Installed Satellite with Organization, Activation key,
+                Content View, Content Host, and Subscriptions.
+
+        :steps:
+            1. Monitor -> Report Templates
+            2. Entitlements -> Generate
+            3. Click the dropdown to select output format
+            4. Submit
+
+        :expectedresults: reports are generated containing all the expected information
+                          regarding Entitlements for each output format.
+
+        :CaseImportance: High
+    """
+    with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+        vm.install_katello_ca()
+        vm.register_contenthost(content_setup[0].label, content_setup[1].name)
+        assert vm.subscribed
+        with session:
+            session.organization.select(content_setup[0].name)
+            session.location.select('Default Location')
```

Comment:
You should drop these two lines and rework your fixture a little. `session` fixture will log in user that already have specific location and organization created and set as default. See https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/conftest.py#L112 . I believe your fixture can depend on `module_org`, and this will be the same organization as `module_user` depends on.

Reply:
@mirekdlugosz can he get the organization/location id's from the session.organization/location objects? or are you recommended he adds module_org and module_loc to the tests' arguments?

Reply:
@JacobCallahan Oh no, `session` fixture returns Airgun `Session` class instance, and we don't store references to organization/location there.

I think `content_setup` fixture should pull in `module_org`. Simplified:
```python
@pytest.fixture(scope='module')
def content_setup(module_org):
    rh_repo_id = enable_rhrepo_and_fetchid(org_id=module_org.id, ...)
```
That should do the trick.